src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogHttpClient.java

```diff
@@ -588,4 +612,9 @@ private String getJenkinsVersion(){
         return this.jenkinsVersion;
     }
 
+    private static boolean isCollectBuildLogEnabled(){
```

Comment:
I don't like that we need to access the config from here.

Reply:
This is also currently in DogstatsdClient. I'm not sure the best way to check for logs being enabled.